crates/holochain/src/conductor/conductor.rs

```diff
@@ -2963,6 +2962,92 @@ mod misc_impls {
             Ok(action_hash)
         }
 
+        /// Get capability grant info for a set of App cells including revoked capabality grants
+        pub async fn capability_grant_info(
+            &self,
+            cell_set: &HashSet<CellId>,
+            include_revoked: bool,
+        ) -> ConductorApiResult<AppCapGrantInfo> {
+            let mut hash_map: HashMap<CellId, Vec<CapGrantInfo>> = HashMap::new();
+            let grant_query: ChainQueryFilter = ChainQueryFilter::new()
```

Comment:
This is a direct initialization. Even without type inferral the type is clear.
```suggestion
            let grant_query = ChainQueryFilter::new()
```